lib/PuppeteerSharp.Tests/BrowserContextTests/BrowserContextTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PuppeteerSharp.Tests.BrowserContextTests
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class BrowserContextTests : PuppeteerBrowserBaseTest
+    {
+        public BrowserContextTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task ShouldHaveDefaultContext()
+        {
+            Assert.Single(Browser.BrowserContexts());
+            var defaultContext = Browser.BrowserContexts()[0];
+            Assert.False(defaultContext.IsIncognito);
+            var exception = await Assert.ThrowsAsync<PuppeteerException>(
+                () => defaultContext.CloseAsync());
```

Comment:
This can me expressed as:
`var exception = await Assert.ThrowsAsync<PuppeteerException>(defaultContext.CloseAsync);`